rpm/xrootd-hdfs.spec

```diff
@@ -1,5 +1,5 @@
 Name: xrootd-hdfs
-Version: 2.0.1
+Version: 2.0.2
```

Comment:
Bumping the Version number is in line with what @bbockelm requested, so I won't ask you to change it back.  But normally a spec-only change would only warrant bumping the Release number.  Bundling the spec file in the upstream repo perhaps makes it a bit trickier to de-couple packaging-only changes from non-packaging changes in releases, but `osg-build` does have support for tags like `v2.0.1-2`, where the `-2` tag indicates only spec changes were made.